docs/src/guide/installation.md

```diff
@@ -0,0 +1,43 @@
+# Installation
+
+DFTK is not yet registered in the [General](https://github.com/JuliaRegistries/General)
+registry of Julia.
+Instead you can obtain it from
+the [MolSim](https://github.com/JuliaMolSim/MolSim.git) registry,
+which contains a bunch of packages related to performing molecular simulations in Julia.
+At least **Julia 1.3** is required.
+
+First add `MolSim` to your installed registries. For this use
+```
+] registry add https://github.com/JuliaMolSim/MolSim.git
+```
+from a Julia command line.
+Afterwards you can install DFTK like any other package in Julia:
+```
+] add DFTK
+```
+or if you like the bleeding edge:
+```
+] add DFTK#master
+```
+
+Some parts of the code require a working Python installation with the libraries
+[`pymatgen`](https://pymatgen.org/) and [`spglib`](https://atztogo.github.io/spglib/).
+Check out which version of python is used by the
+[PyCall.jl](https://github.com/JuliaPy/PyCall.jl) package.
+You can do this for example with the Julia commands
+```julia
+using PyCall
+PyCall.python
+```
+Then use the corresponding package manager (usually `apt`, `pip`, `pip3` or `conda`)
+to install aforementioned libraries, for example
+```
+pip install spglib pymatgen
```

Comment:
pip3 is most likely to be the one used, no? Also `sudo`?

Reply:
depends on the os